lms/djangoapps/onboarding/models.py

```diff
@@ -23,6 +23,7 @@
 register(User, app=__package__, table_name='auth_historicaluser')
 register(UserProfile, table_name='auth_historicaluserprofile')
 
+models.CharField.register_lookup(models.functions.Length, 'length')
```

Comment:
How about to directly do this at some generic place so that we can use this everywhere?

Reply:
Don't have any suggestion about the place, but we can figure it out in a call.

Reply:
@philanthropy-u/reviewers please suggest something